src/qt/optionsmodel.h

```diff
@@ -48,6 +49,9 @@ class OptionsModel : public QAbstractListModel
         ThreadsScriptVerif,     // int
         DatabaseCache,          // int
         SpendZeroConfChange,    // bool
+        AddressTypeLegacy,      // bool
+        AddressTypeP2SH,        // bool
```

Comment:
Maybe for constancy reason, change that to `AddressTypeP2SHSegWit`?

Reply:
Agree.

Reply:
Fixed.